omero/sysadmins/unix/server-install-homebrew.rst

```diff
@@ -41,155 +51,196 @@ running::
     $ javac -version
     javac 1.8.0_51
 
-Requirements
-------------
 
-.. _`Homebrew wiki`: https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Installation.md
+OS X Basics
+-----------
 
-All the requirements for OMERO will be installed under :file:`/usr/local`. See also: Installation instructions on the `Homebrew wiki`_.
+In order to develop on OMERO, we recommend you ensure you have your Mac setup for
+development. The first step to achieving this is to create a :file:`.bash_profile` file in the
+root directory of your user folder.
 
-Install Homebrew and make sure :file:`/usr/local/bin` is prepended to your
-:envvar:`PATH`::
+To create a :file:`.bash_profile` from terminal, if one does not already exist::
 
-    $ ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
-    $ export PATH=/usr/local/bin:$PATH
+    $ touch ~/.bash_profile
 
-Update Homebrew and run 'brew doctor' to fix potential issues beforehand::
+To open your :file:`.bash_profile` in a text editor, such as the built-in TextEdit app, use::
 
-    $ brew update
-    $ brew doctor
+    $ open -a TextEdit.app ~/.bash_profile
 
-Install git if not already present::
+.. note::
+   If you want to see changes to your :file:`.bash_profile` take effect without restarting
+   OS X, run::
 
-    $ brew list | grep "\bgit\b" || brew install git
+   $ source ~/.bash_profile
 
-Install PostgreSQL database server::
+Requirements
+------------
+
+1. Open a command-line terminal and install git if not already present::
+
+    $ brew install git
+
+2. Install PostgreSQL database server::
 
-    $ export LANG=${LANG:-en_US.UTF-8}
-    $ export LANGUAGE=${LANGUAGE:-en_US:en}
     $ brew install postgresql
 
-.. _`Homebrew and Python`: https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Homebrew-and-Python.md
+   To ensure PostgreSQL uses UTF-8 encoding, open your bash profile and 
+   add the following enviornment variables::
+
+    export LANG=en_US.UTF-8
+    export LANGUAGE=en_US:en
+
+3. OMERO depends on Ice 3.6 and unfortunately does not run with 
```

Comment:
``OMERO depends on Ice 3.6 and unfortunately does not run with the latest version of Ice at this time (Ice 3.7.3). To obtain Ice 3.6, we need to add a *tap* to Homebrew::`` 
replace with 
``OMERO depends on Ice but does not run with the latest version of Ice at this time (Ice 3.7.3). To obtain the version of Ice recommended for OMERO (Ice 3.6) we need to add a "tap" to Homebrew::``
 (do not like the "unfortunately", and OMERO does not depend on Ice 3.6 specifically, we support also Ice 3.5)

Reply:
```
brew info ice
ice: stable 3.7.0 (bottled)
```